ggshield/utils/git_shell.py

```diff
@@ -445,3 +445,16 @@ def parse_name_status_patch(patch: str) -> Dict[Path, Filemode]:
 def read_git_file(ref: str, path: Path, wd: Optional[Union[str, Path]] = None) -> str:
     # Use as_posix to handle git and Windows
     return git(["show", f"{ref}:{path.as_posix()}"], cwd=wd)
+
+
+def get_mr_commits(
```

Comment:
This function lists all commits not in the current branch, I think we can give it a name not linked to GitLab (MR is very GitLab-only). May `get_commits_not_in_branch()`?